bim2sim/workflow.py

```diff
@@ -272,6 +272,36 @@ def update_from_config(self, config):
                     'Choose 0.3m as a default value.',
         for_frontend=True
     )
+
+    group_unidentified = WorkflowSetting(
+        default='fuzzy',
+        choices={
+            'fuzzy': 'Use fuzzy search to find name similarities',
+            'name': 'Only group elements with exact same name'
+        },
+        description='To reduce the number of decisions by user to identify '
+                    'elements which can not be identified automatically by the '
+                    'system, you can either use simple grouping by same name of'
+                    ' IFC element or fuzzy search to group based on'
+                    ' similarities in name.',
+        for_frontend=True
+    )
+    fuzzy_threshold = WorkflowSetting(
+        default=0.7,
```

Comment:
good idea, but I think this should be solved in another issue as this would also be useful for e.g. the already existing setting `max_wall_thickness`.

I opened an issue with a solution apporach [here](https://github.com/BIM2SIM/bim2sim/issues/506)